### PR TITLE
6.5.0-beta1 Changelog

### PR DESCRIPTION
**Pull request by sunker**

### CODE REVIEW

**Review by aknuds1 — APPROVED** [review]
Looks good AFAICT, apart from a formatting nit.

**Review by marefr — CHANGES_REQUESTED** [review]
Please wait with the merge

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+### Features / Enhancements
+* **CloudWatch**: Support dynamic queries using dimension wildcards [#20058](https://github.com/grafana/grafana/issues/20058), [@sunker](https://github.com/sunker)
+* **CloudWatch**: Stop using GetMetricStatistics and use GetMetricData for all time series requests [#20057](https://github.com/grafana/grafana/issues/20057), [@sunker](https://github.com/sunker)
+* **CloudWatch**: Convert query editor from Angular to React [#19880](https://github.com/grafana/grafana/issues/19880), [@sunker](https://github.com/sunker)
+* **CloudWatch**: Convert config editor from Angular to React [#19881](https://github.com/grafana/grafana/issues/19881), [@shavonn](https://github.com/shavonn)
+* **CloudWatch**: Improved error handling when throttling occurs [#20348](https://github.com/grafana/grafana/issues/20348), [@sunker](https://github.com/sunker)
+* **CloudWatch**: Deep linking from Grafana panel to CloudWatch console [#20279](https://github.com/grafana/grafana/issues/20279), [@sunker](https://github.com/sunker)
+* **CloudWatch**: Add Grafana user agent to GMD calls [#20277](https://github.com/grafana/grafana/issues/20277), [@sunker](https://github.com/sunker)
+
+
```
[the remark this answers is not in the document]
Comment (sunker) [reply CHANGELOG.md]: Yes!

[the remark this answers is not in the document]
Comment (aknuds1) [reply CHANGELOG.md]: Thanks @sunker :)

[the remark this answers is not in the document]
Comment (sunker) [reply CHANGELOG.md]: Thank you! :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -139,7 +211,7 @@
 * **Provisioning**: Allow escaping literal '$' with '$$' in configs to avoid interpolation. [#18045](https://github.com/grafana/grafana/pull/18045), [@kylebrandt](https://github.com/kylebrandt)
 * **TimePicker**: Fixes hiding time picker dropdown in FireFox. [#19154](https://github.com/grafana/grafana/pull/19154), [@hugohaggmark](https://github.com/hugohaggmark)
 
-## Breaking changes
+##  ing changes
```
Comment (daniellee) [inline CHANGELOG.md]: This doesn't look right

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+
+* **CloudWatch**: In previous versions of Grafana, there were partial support for using multi template variables as dimension values. When a multi template variables is being used for dimension values in Grafana 6.5, a [search expression](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/using-search-expressions.html) will be generated. In the GetMetricData API, expressions are limited to 1024 characters, so it might be the case that this limit is reached when a multi template variable is being used.
```
Comment (daniellee) [inline CHANGELOG.md]: - "there was partial support" - replace was with were
- "When a multi template variable" - drop the s

**Review by daniellee — APPROVED** [review]
LGTM

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**